azure-pipelines-foundation.yml

```diff
@@ -0,0 +1,60 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+- azuredevops_pipeline
+
+variables:
+  azureSubscription: MSDN_nicolasyuen
```

Comment:
Having this as a variable and some documentation on the setup of the pipeline, might be a better final state.

Understand this is for testing, but would fail when anyone who didn't have access to your subscription ran it first time if they weren't across how pipelines works.

Reply:
Agreed, moving the variable to the pipeline itself